modules/tools/rode/variables.tf

```
@@ -1,15 +1,36 @@
-variable "enable_rode" {
-  default = true
+variable "namespace" {}
+
+variable "rode_ingress_hostname" {}
+
+variable "ui_ingress_hostname" {
+  description = "Leave blank to disable rode-ui"
+  default     = ""
 }
 
-variable "rode_service_account_arn" {}
+variable "ingress_class" {
+  default = "toolchain-nginx"
+}
 
-variable "namespace" {}
+variable "oidc_issuer_url" {
+  description = "Leave blank to disable OIDC"
+  default     = ""
+}
+variable "oidc_issuer_client_id" {
```

Nitpick, but I'd just call this client id/client secret. 

```suggestion
variable "oidc_client_id" {
```